graphene_django/management/commands/graphql_schema.py

```diff
@@ -60,8 +60,10 @@ def save_graphql_file(self, out, schema):
 
     def get_schema(self, schema, out, indent):
         schema_dict = {"data": schema.introspect()}
-        if out == "-":
+        if out == "-" or out == "-.json":
             self.stdout.write(json.dumps(schema_dict, indent=indent, sort_keys=True))
+        elif out == "-.graphql":
+            print(print_schema(schema))
```

Comment:
This should use stdout instead of print.

Reply:
Changed, my friend.